warehouse/packaging/models.py

```diff
@@ -175,6 +176,17 @@ def documentation_url(self):
 
         return request.route_url("legacy.docs", project=self.name)
 
+    @property
+    @functools.lru_cache()
```

Comment:
_smart_! my only concern here is that _technically_ this can grow unbounded without a `maxsize` parameter attached, and I'm uncertain exactly how long these will linger in the garbage collector... any ideas?

Reply:
I guess it gets GC'd with the object... so per request?

Reply:
> I guess it gets GC'd with the object

Nope, it doesn't! Good catch.